code-studio/src/js/code-studio.js

```diff
@@ -16,6 +16,8 @@ require('./videos');
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
+window.ColorPicker = require('react-color').default;
```

Comment:
> I guess my question is, why isn't React imported explicitly in all the jsx files?

I believe we still have a few places where React is being used by dashboard code, which is not CommonJS compatible and depends on global React being available.  Global React naturally conflicts with any explicit imports we do in our JSX files - so we don't explicitly import it anywhere.  Once any remaining dashboard uses are cleaned up I think we can start requiring it, though the apps/code-studio split might still cause trouble.

@Bjvanminnen do you know if we could/should use browserify-shim with react here, (changing the react-color package's internal react import to point at our global instance)?